packages/application-to-register-webapp/src/utils/constants.js

```diff
@@ -101,8 +101,11 @@ const LANDOWNERS = 'landowners'
 const LANDOWNER_CONSENT = 'land/landowner-consent'
 const LANDOWNER_CONSENT_KEY = 'landowner-consent'
 const CHECK_OWNERSHIP_DETAILS = 'land/check-ownership-details'
+const REGISTRATION_TASK_DETAILS = 'registrationTaskDetails'
+const DEFAULT_REGISTRATION_TASK_STATUS = 'NOT STARTED'
 const REFERRAL_PAGE_LIST = [
-  `/${LEGAL_AGREEMENT_SUMMARY}`
+  `/${LEGAL_AGREEMENT_SUMMARY}`,
+  REGISTRATION_TASK_DETAILS
```

Comment:
I'm not sure the need to add REGISTRATION_TASK_DETAILS here, as it shouldn't be caught at all by the referer logic.  And also please note that with BNGP-190 this object no longer exists for the referer routing

Reply:
Can we have a discussion on the referrer routing. I add it here to have a list of pages that can be considered as referral candidates. Not all pages can be referred from.

Reply:
See here https://github.com/DEFRA/biodiversity-net-gain-service/blob/82b2a7572f26936d13ed9e7d559f2e41b21b1e20/packages/application-to-register-webapp/src/utils/constants.js#L267

Reply:
Same concept. I agree that we don't need REGISTRATION_TASK_DETAILS.
Is your checkRoutes array merged to master?

Reply:
You are describing what I have implemented. Have you looked at the code?